### PR TITLE
Blog posts/monte carlo power analysis

### PR DESCRIPTION
**Pull request by danielnee**
New blog post by the Data Science team. 

Only thing I've noticed is that is doesn't like display three authors. At the top, it doesn't add a comma between 1st and 2nd author. At the bottom the 3rd author's 'About' is not aligned like the rest.

screenshot 2018-11-29 at 09 39 36
screenshot 2018-11-29 at 09 39 54

### CODE REVIEW

File: _posts/2018-11-29-monte-carlo-power-analysis.md [hunk 1]
```diff
@@ -0,0 +1,334 @@
...
+
+## What is statistical power?
+
+We use frequentist hypothesis testing approaches to analyse the impact of any experiment. The statistical power of an experiment for a given alternative hypothesis is the probability we will get a statistically significant result (reject the null hypothesis) when the alternative is true for any one realisation of the experiment:
+
+$$power = 1 - \beta = P(\textrm{Reject }H_0|H_1\textrm{ is true})$$
```
Comment (marianeum) [inline _posts/2018-11-29-monte-carlo-power-analysis.md]: maybe the formulas could be rendered as code? e.g. 
```
$$power = 1 - \beta = P(\textrm{Reject }H_0|H_1\textrm{ is true})$$
```

(use triple back tips around the code)

File: _posts/2018-11-29-monte-carlo-power-analysis.md [hunk 2]
```diff
@@ -0,0 +1,334 @@
...
+
+A result is determined to be statistically significant if the observed effect was unlikely to have been seen by random chance. How unlikely is determined by the significance level, $$\alpha$$, that is the likelihood of rejecting the null hypothesis given it is true:
```
Comment (marianeum) [inline _posts/2018-11-29-monte-carlo-power-analysis.md]: the variables could also be rendered as code maybe - e.g. `$$\alpha$$` (surround with a single back tick)

File: _posts/2018-11-29-monte-carlo-power-analysis.md (hunk 1 above)
Comment (danielnee) [reply _posts/2018-11-29-monte-carlo-power-analysis.md]: For the moment I've tried to be consistent with the previous blog post https://deliveroo.engineering/2018/10/22/how-to-experiment-rapidly-without-losing-rigour.html

File: _posts/2018-11-29-monte-carlo-power-analysis.md (hunk 2 above)
Comment (danielnee) [reply _posts/2018-11-29-monte-carlo-power-analysis.md]: As above

**Comment by danielnee** [comment]
I found a bug in the code so please hold off merging

**Comment by danielnee** [comment]
@kapowaz @jphastings @Crunch09 Any chance you could look at the rendering issues with the third author?

If it is a lot of work - I would be tempted to push it live with the issue. It seems very minor

**Pull request merged**